store/rootmulti/store.go

```diff
@@ -905,6 +905,30 @@ func (rs *Store) buildCommitInfo(version int64) *types.CommitInfo {
 	}
 }
 
+// RollbackToVersion delete the versions after `target` and update the latest version.
+func (rs *Store) RollbackToVersion(target int64) int64 {
+	if target < 0 {
```

Comment:
the input target should be always greater than 1 right? the State.InitialHeight will be 1.

Reply:
yes, I think so, just for safety here.